backend/src/main.ts

```diff
@@ -11,5 +14,35 @@ async function bootstrap() {
     new FastifyAdapter(),
   );
   await app.listen(8080, "0.0.0.0");
+
+  app.use(urlencoded({ extended: true }));
+  app.use(json());
+  const { schema } = app.get(GraphQLSchemaHost);
+  const openApi = OpenAPI({
+    schema,
+    info: {
+      title: "NFT4Charity API",
+      version: "3.0.0",
+    },
+    servers: [
+      {
+        url: "http://localhost:8080/api",
```

Comment:
change this to `/rest` nhé?

Reply:
Updated!
<img width="421" alt="Screen Shot 2021-09-21 at 22 27 58" src="https://user-images.githubusercontent.com/60747384/134200423-c7effc1b-4378-4808-9bb3-823e3264b1de.png">

Reply:
Without `yarn.lock`, an error occurs when building image.

<img width="1440" alt="Screen Shot 2021-09-21 at 22 33 26" src="https://user-images.githubusercontent.com/60747384/134201377-9eb1bf18-f17d-4f10-8d8e-6e9bf539f64c.png">